client/build_linux_test.go

```diff
@@ -0,0 +1,258 @@
+// +build linux
+
+package client
+
+import (
+	"bufio"
+	"context"
+	"os"
+	"syscall"
+	"testing"
+
+	"github.com/containerd/console"
+	"github.com/moby/buildkit/client/llb"
+	"github.com/moby/buildkit/frontend/gateway/client"
+	"github.com/moby/buildkit/solver/pb"
+	"github.com/moby/buildkit/util/testutil/integration"
+	"github.com/pkg/errors"
+	"github.com/stretchr/testify/require"
+)
+
+func TestClientGatewayIntegrationLinux(t *testing.T) {
+	integration.Run(t, []integration.Test{
+		testClientGatewayContainerPID1Tty,
+		testClientGatewayContainerExecTty,
+	}, integration.WithMirroredImages(integration.OfficialImages("busybox:latest")))
+}
+
+// testClientGatewayContainerPID1Tty is testing that we can get a tty via
+// a container pid1, executor.Run
+func testClientGatewayContainerPID1Tty(t *testing.T, sb integration.Sandbox) {
+	requiresLinux(t)
```

Comment:
we need the `+build linux` otherwise the test wont compile on windows due to `container/console` dep.  So, yeah, I think I can remove the `requiresLinux`